app/src/main/java/com/sdp/movemeet/utility/LocationFetcher.java

```diff
@@ -31,9 +31,11 @@ public static void fetchLastLocation(SupportMapFragment supportMapFragment, Fuse
         task.addOnSuccessListener(new OnSuccessListener<Location>() {
             @Override
             public void onSuccess(Location location) {
-                currentLocation = location;
-                Toast.makeText(supportMapFragment.getActivity(), currentLocation.getLatitude() + ", " + currentLocation.getLongitude(), Toast.LENGTH_SHORT).show();
-                supportMapFragment.getMapAsync(callback);
+                if (location != null) {
```

Comment:
What's the purpose of this change?

Reply:
For some reason, I would get errors when running tests, because the location returned by the successful task was somehow null

Reply:
Ok, that makes sense!